lib/stat_tracker.rb

```diff
@@ -59,4 +59,77 @@ def read_game_teams_csv(location_path)
       end
       game_team_data
    end
+
+   def count_of_teams
+      @data_teams.count  
+   end
+
+   def best_offense
+      convert_team_id_to_name(highest_average_team_id(team_stats))
+   end
+
+   def worst_offense
+      convert_team_id_to_name(lowest_average_team_id(team_stats))
+   end
+
+   def highest_scoring_visitor
+      convert_team_id_to_name(highest_average_team_id(team_stats_hoa("away")))
+   end
+
+   def highest_scoring_home_team
+      convert_team_id_to_name(highest_average_team_id(team_stats_hoa("home")))
+   end
+
+   def lowest_scoring_visitor
+      convert_team_id_to_name(lowest_average_team_id(team_stats_hoa("away")))
+   end
+
+   def lowest_scoring_home_team
+      convert_team_id_to_name(lowest_average_team_id(team_stats_hoa("home")))
+   end
+
+   #helper methods
+
+   def convert_team_id_to_name(team_id)
+      team_name = "0"
+      @data_teams.each do |team| 
```

Comment:
I think you can use the enumerable method `.find` here and reduce the code. It def can be done later, just an idea to reduce the amount of code.